d1985r3.md

```diff
@@ -468,6 +468,19 @@ Z<int> z1; // ok, decltype(z1)::value = int
 Z<1> z; // error, alias<1>::value is not a type
 ```
 
+## Impacts on the function template overloading
+
+Universal template arguments impact template overloading:
+
+```cpp
+template<template auto X> const char* kind_name() { return "type"; }
+template<template<template auto...> X> const char* kind_name() { return "template"; }
+template<auto X> const char* kind_name() { return "value"; }
+```
+
+Function templates taking a universal template argument is considered
+a worse match than the one taking a specific template parameter kind.
+
```

Comment:
Maybe add something about variadics being less specific than a fixed count anykind (derived from how concept constrained variadic is less specific than uncontrained fixed count). See PDF from Sep 8.